lib/ecto/adapters/sql.ex

```diff
@@ -1377,21 +1425,37 @@ defmodule Ecto.Adapters.SQL do
 
   @repo_modules [Ecto.Repo.Queryable, Ecto.Repo.Schema, Ecto.Repo.Transaction]
 
-  defp last_non_ecto([{mod, _, _, _} | _stacktrace], repo, last)
+  @doc """
+  Receives a stacktrace, and return the first N items before Ecto entries
+
+  This function is used by default in the `:log_stacktrace_info` config, with
```

Comment:
```suggestion
  This function is used by default in the `:log_stacktrace_mfa` config, with
```
I think it should be this but I might be getting confused